selfdrive/car/honda/carstate.py

```diff
@@ -235,19 +235,24 @@ def update(self, cp):
     else:
       self.park_brake = 0  # TODO
       self.brake_hold = 0  # TODO
+   
       self.main_on = cp.vl["SCM_BUTTONS"]['MAIN_ON']
 
     self.cruise_speed_offset = calc_cruise_offset(cp.vl["CRUISE_PARAMS"]['CRUISE_SPEED_OFFSET'], self.v_ego)
     self.gear_shifter = parse_gear_shifter(can_gear_shifter, self.CP.carFingerprint)
 
     self.pedal_gas = cp.vl["POWERTRAIN_DATA"]['PEDAL_GAS']
     # crv doesn't include cruise control
-    if self.CP.carFingerprint != CAR.CRV:
+    if self.CP.carFingerprint != CAR.CRV or CAR.ACURA_RDX:
```

Comment:
@vanillagorillaa this should be 
```if CS.CP.carFingerprint not in (CAR.CRV, CAR.ACURA_RDX)```

actually, I would invert the check as follow:
```
if CS.CP.carFingerprint in (CAR.CRV, CAR.ACURA_RDX):
  self.car_gas = self.pedal_gas
else:
  self.car_gas = cp.vl["GAS_PEDAL_2"]['CAR_GAS']